modules/common/react/lib/theming/styled.ts

```diff
@@ -1,5 +1,5 @@
 import {default as emotionStyled, CreateStyled, Interpolation, CSSObject} from '@emotion/styled';
-import {CanvasTheme, ContentDirection} from '@workday/canvas-kit-labs-react-core';
+import {CanvasTheme, ContentDirection} from '@workday/canvas-kit-react-common';
```

Comment:
Another benefit of moving it to common is how nicely it plays with importing other things together